lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -46,9 +65,19 @@ all_afs:
   config_get_token_append: '/^address-family (\S+) (\S+)$/'
 
 client_to_client:
-  config_get_token_append: '/^client-to-client reflection$/'
-  config_set_append: '<state> client-to-client reflection'
-  default_value: true
+  cli_nexus:
+    config_get_token_append: '/^client-to-client reflection$/'
+    config_set_append: '<state> client-to-client reflection'
+    default_value: true
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp client-to-client reflection disable'
+    config_get: 'show running-config router bgp | in bgp client-to-client'
+    config_get_token: '/^router bgp <asnum>$/'
+    config_get_token_append: '/^bgp client-to-client reflection disable$/'
```

Comment:
It can only appear in one place, so I just gripped for bgp client-to-client directly